Makefile

```diff
@@ -179,7 +176,7 @@ else ifeq ($(ANDROID_ARCH), all)
 # native libs, 32 bit Intel devices will just emulate ARM.
 # DO NOT ADD x86 TO THIS LIST!!
   ANDROID_ARCH_JAVA := arm64-v8a x86_64
-  ANDROID_ARCH_GOMOBILE := android/arm64,android/amd64
+  ANDROID_ARCH_GOMOBILE := android
```

Comment:
"By default, -target=android builds shared libraries for all supported instruction sets" https://pkg.go.dev/golang.org/x/mobile/cmd/gomobile

Reply:
So basically, Now we will generate go lib for all arch, But include 64 bit while building correct?